sources/packages/web/src/views/institution/locations/programs/LocationProgramOffering.vue

```diff
@@ -155,15 +154,31 @@ export default {
             ...programDetails,
           };
         }
+        /**
+         * The property clientType is populated for institution because
+         * the form.io for education program offering has a logic at it's root level panel
+         * to disable all the form inputs when clientType is not institution.
+         * The above mentioned logic is added to the panel of the form to display the
+         * form as read-only for ministry(AEST) user and also allow the hidden component values
+         * to be calculated.
+         *! If a form.io is loaded with readOnly attribute set to true, then the restricts
+         *! hidden components to calculate it's value by design.
+         */
+        initialData.value.clientType = ClientIdType.Institution;
```

Comment:
Can we just assign it the `AuthService.shared.authClientType` directly and have it always assigned independent of the client to keep it consistent?